docs/endpoints/getGJLevelScores211.md

```diff
@@ -28,7 +26,7 @@ Upload and query user levelBoard record.
 
 **s3** - seconds + 4085
 
-**s4** - 1482 * 2 + (jumps + 3991) * (percentage + 8354) + ((seconds + 4085) ** 2) - 50028039
+**s4** - 1482 * 2 + (jumps + 3991) * (percentage + 8354) + ((seconds + 4085) ^ 2) - 50028039
```

Comment:
as I said, highlight calculations using ` `` ` and add a brief description before them. these are docs, not calculations. this also applies for all the other cases you only added calculations

Reply:
Also, perhaps use ``1482 * (has played + 1) + ...`` as it was done before.